client/src/components/GameImage/index.js

```diff
@@ -74,7 +83,8 @@ GameImage.propTypes = {
   id: PropTypes.number.isRequired,
   handleSelected: PropTypes.func.isRequired,
   transitionSpeed: PropTypes.string,
-  selected: PropTypes.bool
+  selected: PropTypes.bool,
+  username: PropTypes.string
```

Comment:
I think this should be required?
```suggestion
  username: PropTypes.string.isRequired
```